### ISSUE TITLE
Add authorization request header while performing tests.

### ISSUE THREAD

**Issue by burntcarrot** [burntcarrot/blaezi#6]
Currently, we have the `-auth` flag, but there is no implementation for adding the auth string while performing the tests.

We can preferably add the following to at [line 68](https://github.com/burntcarrot/blaezi/blob/main/inspector.go#L68):

```go
if inspector.auth != "" {
	req.Header.Add("Authorization", inspector.auth)
}
```

**Comment by krishnaindani** [comment]
Looks like simple change. Can I work on this as well?

**Comment by burntcarrot** [comment]
> Looks like simple change. Can I work on this as well?

Yes, but let's resolve #7 first.

**Comment by krishnaindani** [comment]
sure thing. I have added a PR there

**Comment by burntcarrot** [comment]
Resolved via #10.
